### ISSUE TITLE
Announce new presentation

### ISSUE THREAD

**Issue by antobinary** [bigbluebutton/bigbluebutton#14141]
**Describe the bug**
When a new presentation is shared (or switching to a different [preuploaded] one), there should be a notification for screen reader users.

**To Reproduce**
1. Join a session as presenter and upload two presentations
2. Join as a viewer in the session (with screen reader ON)
3. The presenter switches the current presentation

**Expected behavior**
The viewer should be notified in a way that is picked up by the screen reader.

**Actual behavior**
No notification about the switch of presentation.

**Screenshots**

**BBB version:**
BigBlueButton 2.4.2

**More**
First reported in https://github.com/bigbluebutton/bigbluebutton/issues/11448 by @audioses
